hnn_core/tests/test_gui.py

```diff
@@ -716,9 +730,12 @@ def test_gui_upload_csv_simulation(setup_gui):
                 ['dpls'][0].data['L5']) == data_lengh)
 
 
-def test_gui_add_tonic_input(setup_gui):
+def test_gui_add_tonic_input():
     """Test if gui add different type of drives."""
-    gui = setup_gui
+    gui = HNNGUI()
+    _ = gui.compose()
```

Comment:
Changed this because the setup_gui already has a tonic and we want to test adding a tonic.